dependencies.yaml

```diff
@@ -225,11 +226,23 @@ dependencies:
           - &gmock gmock>=1.13.0
           # Hard pin the patch version used during the build. This must be kept
           # in sync with the version pinned in get_arrow.cmake.
-          - libarrow==12.0.1.*
+          # Temporarily building a specific build string in order to choose one
+          # that is compatible with the aws-sdk-cpp version required above
```

Comment:
This comment (and the one above) belong next to the `libarrow` pinning, not next to `librdkafka`.